server/routes/verifyAddress.ts

```diff
@@ -359,13 +453,18 @@ const verifyAddress = async (models: DB, req: Request, res: Response, next: Next
     });
     req.login(user, (err) => {
       if (err) return next(err);
+      const mixpanelData: MixpanelLoginPayload = {
```

Comment:
I recommend just dropping the object literal into `mixpanelTrack` so each tracking event is just one line. The inclusion of `event` will make it automatically "know" which struct type it is, if we do a little typescript magic...